app/services/database_manager.py

```diff
@@ -0,0 +1,231 @@
+from qdrant_client import QdrantClient
+from qdrant_client.http import models as rest
+from sentence_transformers import SentenceTransformer
+import fitz
+
+# Configuration for Qdrant client
+client = QdrantClient(host="localhost", port=6333)
+
+# Sentence-BERT model
+model = SentenceTransformer('sentence-transformers/all-MiniLM-L6-v2')
```

Comment:
As a question, how would you handle artifact management with this model? e.g , how would you ensure this model is always available for download?